canonical/juju-dashboard
# WD-5283 - Change ButtonGroup to Vanilla Segmented Controls

## Done

- Renamed ButtonGroup to SegmentedControls.
- Replaced custom button group styling with Segmented Controls classes.

## QA

- Go to the model list.
- Check that the Status/Cloud/Owner controls look like the segment controls: https://vanillaframework.io/docs/patterns/segmented-control.

## Details

This is required so that the action log/audit log buttons match the style and alignment of the filters that will be added.
https://warthogs.atlassian.net/browse/WD-5283

## Screenshots

<img width="840" alt="Screenshot 2023-07-26 at 1 50 51 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/cd89846e-6200-40c1-84d2-4b5fbf0f558d">
